[CI:DOCS] docs: specify `git://` protocol is not supported for github hosted repo (#4179)
containers/buildah: docs/buildah-build.1.md

  This will clone the specified GitHub repository from the URL and use it as context. The Containerfile or Dockerfile at the root of the repository is used as the context of the build. This only works if the GitHub repository is a dedicated repository.

  buildah build https://github.com/scollier/purpletest

>   Note: Github does not support using `git://` for performing `clone` operation due to recent changes in their security guidance (https://github.blog/2021-09-01-improving-git-protocol-security-github/).Use a `https://` URL if the source repository is hosted on Github.

Reviewer: ```suggestion
  Note: Github does not support using `git://` for performing `clone` operation due to recent changes in their security guidance (https://github.blog/2021-09-01-improving-git-protocol-security-github/).  Use an `https://` URL if the source repository is hosted on Github.
```

>   Note: Github does not supports using `git://` for performing `clone` operation due to recent changes in their security protocol ( https://github.blog/2021-09-01-improving-git-protocol-security-github/ ) therefore it is advised to use `https://` if the source repository is hosted on Github.

Reviewer: ```suggestion
  Note: Github does not support using `git://` for performing `clone` operation due to recent changes in their security guidance (https://github.blog/2021-09-01-improving-git-protocol-security-github/).  You must use a `https://` URL if the source repository is hosted on Github.
```

Reviewer 2: Remove You must, and just say `Use a https:// ...`

@TomSweeneyRedHat hates us using You within man pages.

Author: amended.
